client/my-sites/current-site/style.scss

```diff
@@ -61,46 +61,34 @@
 .current-site__switch-sites {
 	background: lighten( $gray, 30% );
 	border-bottom: 1px solid lighten( $gray, 25% );
-	font-size: 13px;
 	display: block;
-	padding: 8px 8px 8px 40px;
+	padding: 8px;
 	color: darken( $gray, 20% );
```

Comment:
Should we remove this? The color is already defined in the button styles for both .is-compact` and 'is-borderless`.